### PR TITLE
Make order fund

### PR DESCRIPTION
**Pull request by jennazenk**

### CODE REVIEW

**Comment by jennazenk** [comment]
Not ready to merge yet; unit tests not passing.

**Comment by jennazenk** [comment]
@schmidsi ready to merge into 0.3 w all main functionality implemented and tested. Was thinking perhaps we should merge 0.3 into master also. 

Still have to do: test redeem, and implement all interface functions that we don't have yet. But I think right now what we have is already enough to make d-portal fully functional.

File: lib/fund/transactions/makeOrderFromFund.js [hunk 1]
```diff
@@ -3,38 +3,53 @@ import BigNumber from "bignumber.js";
...
     getAddress(buyWhichToken),
+    // sellHowMuch,
+    // buyHowMuch,
     toProcessable(sellHowMuch, sellWhichToken),
     toProcessable(buyHowMuch, buyWhichToken),
     { from },
-  );
+  ];
+
+  // const gasEstimation = await fundContract.makeOrder.estimateGas(
```
Comment (schmidsi) [inline lib/fund/transactions/makeOrderFromFund.js]: Why is this commented out? If it works, leave it in. If not, remove it completely.

File: lib/version/transactions/setupFund.js [hunk 2]
```diff
@@ -26,7 +25,9 @@ const setupFund = async (portfolioName, from = setup.defaultAccount) => {
     managementReward,
     performanceReward,
     participation,
+    // "0xab10740b761952f9cf6f0d428b48a71bc963cdfa",
```
Comment (schmidsi) [inline lib/version/transactions/setupFund.js]: Can we remove this?

**Pull request merged**